sql/alter_table.go

```diff
@@ -118,6 +118,10 @@ func (p *planner) AlterTable(n *parser.AlterTable) (planNode, error) {
 		return nil, err
 	}
 
+	// TODO(pmattis): This is a hack. Remove when schema change operations work
+	// properly.
```

Comment:
Well, you're about to fix this, right? And didn't you already file an issue about async schema change operations?